extensions/standard-processors/processors/GetFile.cpp

```diff
@@ -56,7 +56,7 @@ core::Property GetFile::BatchSize(
     core::PropertyBuilder::createProperty("Batch Size")->withDescription("The maximum number of files to pull in each iteration")->withDefaultValue<uint32_t>(10)->build());
 
 core::Property GetFile::Directory(
-    core::PropertyBuilder::createProperty("Input Directory")->withDescription("The input directory from which to pull files")->isRequired(true)->supportsExpressionLanguage(true)->withDefaultValue(".")
+    core::PropertyBuilder::createProperty("Input Directory")->withDescription("The input directory from which to pull files")->isRequired(true)->supportsExpressionLanguage(true)
```

Comment:
Could you add a testcase to that covers this? (The processor doesn't start without setting the input dir)